storage/storage.cpp

```diff
@@ -1209,6 +1209,12 @@ StatusAndError Storage::GetNodeStatus(TCountryTreeNode const & node) const
 
   auto groupStatusCalculator = [&result, &allOnDisk, this](TCountryTreeNode const & nodeInSubtree)
   {
+    // Checks if it's a disputed teritory.
+    vector<TCountryTreeNode const *> found;
+    m_countries.Find(nodeInSubtree.Value().Name(), found);
```

Comment:
В следующем коммите сделано похожим образом:
```
  // Returns true if |node.Value().Name()| is a disputed territory and false otherwise.
  auto checkIfDisputed = [this](TCountryTreeNode const & node)
  {
    vector<TCountryTreeNode const *> found;
    m_countries.Find(node.Value().Name(), found);
    return found.size() > 1;
  };
```
Как вариант, можно сделать метод у Storage IsDisputed(). Давай так и поступим.